tools/star.wdl

```diff
@@ -17,24 +17,24 @@ task build_star_db {
             description: "Name for output in compressed, archived format. The suffix `.tar.gz` will be added.",
             common: true
         }
-        SJDB_gtf_chr_prefix: {
+        sjdb_gtf_chr_prefix: {
             description: "prefix for chromosome names in a GTF file (e.g. 'chr' for using ENSEMBL annotations with UCSC genomes)",
             common: true
         }
-        SJDB_gtf_feature_exon: "feature type in GTF file to be used as exons for building transcripts"
-        SJDB_gtf_tag_exon_parant_transcript: "GTF attribute name for parent transcript ID"
-        SJDB_gtf_tag_exon_parent_gene: "GTF attribute name for parent gene ID"
-        SJDB_gtf_tag_exon_parent_gene_name: "GTF attribute name for parent gene name"
-        SJDB_gtf_tag_exon_parent_gene_type: "GTF attribute name for parent gene type"
+        sjdb_gtf_feature_exon: "feature type in GTF file to be used as exons for building transcripts"
+        sjdb_gtf_tag_exon_parant_transcript: "GTF attribute name for parent transcript ID"
+        sjdb_gtf_tag_exon_parent_gene: "GTF attribute name for parent gene ID"
+        sjdb_gtf_tag_exon_parent_gene_name: "GTF attribute name for parent gene name"
+        sjdb_gtf_tag_exon_parent_gene_type: "GTF attribute name for parent gene type"
         use_all_cores: {
             description: "Use all cores? Recommended for cloud environments.",
             common: true
         }
-        genome_chr_bin_N_bits: "=log2(chrBin), where chrBin is the size of the bins for genome storage: each chromosome will occupy an integer number of bins. For a genome with large number of contigs, it is recommended to scale this parameter as min(18, log2[max(GenomeLength/NumberOfReferences,ReadLength)])."
-        genome_SA_index_N_bases: "length (bases) of the SA pre-indexing string. Typically between 10 and 15. Longer strings will use much more memory, but allow faster searches. For small genomes, the parameter `--genomeSAindexNbases` must be scaled down to `min(14, log2(GenomeLength)/2 - 1)`."
-        genome_SA_sparse_D: "suffix array sparsity, i.e. distance between indices: use bigger numbers to decrease needed RAM at the cost of mapping speed reduction."
+        genome_chr_bin_n_bits: "=log2(chrBin), where chrBin is the size of the bins for genome storage: each chromosome will occupy an integer number of bins. For a genome with large number of contigs, it is recommended to scale this parameter as min(18, log2[max(GenomeLength/NumberOfReferences,ReadLength)])."
+        genome_SA_index_n_bases: "length (bases) of the SA pre-indexing string. Typically between 10 and 15. Longer strings will use much more memory, but allow faster searches. For small genomes, the parameter `--genomeSAindexNbases` must be scaled down to `min(14, log2(GenomeLength)/2 - 1)`."
+        genome_SA_sparse_d: "suffix array sparsity, i.e. distance between indices: use bigger numbers to decrease needed RAM at the cost of mapping speed reduction."
```

Comment:
With the caveat that these will be special cases in the lint rules eventually.

Reply:
Along with 
```
        Int out_QS_conversion_add = 0
        Int out_sam_attr_IH_start = 1
        Int out_sam_flag_OR = 0
        Int out_sam_flag_AND = 65535
```

Reply:
Understood and in agreement.